requirements.txt

```diff
@@ -0,0 +1 @@
+pbr>=1.6
```

Comment:
wondering if we should attempt to add ansible back. In the past we had to remove it to avoid issue with upper constraint globally maintained but we don't have that in the independent org now.

https://review.opendev.org/c/openstack/tripleo-ansible/+/668465

Reply:
This can be discussed later. We definitely need the requrements.txt first to make installation pass.